### PR TITLE
Improve documentation for PR associated to 3rd parties libs or framework

### PR DESCRIPTION
**Pull request by Iragne**
@Arne-W I create this possible improvement. I'm happy to discuss about it

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -44,6 +44,11 @@ if you fix an issue, please add 'Fixes #<issue_number> to the short description
 
 ### Push your changes
 
-Push changes to your repo and send a [pull request]
+Push changes to your repo 
+
+if you update Library, framework or third parties 
+Clone your repo in a different folder and compile+run your new code. This process will ensure that you not missing any file.
```
Comment (Arne-W) [inline CONTRIBUTING.md:50]: I am not an native speaker but i would suggest to change the last sentence to:
"This process ensures that you do not forget any file."

**Comment by Iragne** [comment]
Ok yep me too I'm not native sorry

File: CONTRIBUTING.md (hunk 1 above)
Comment (Iragne) [reply CONTRIBUTING.md]: done

**Comment by Arne-W** [comment]
Nothing to be sorry about :smile:

**Pull request merged**